django/publicmapping/redistricting/management/commands/reconfigure_legislative_body_17_districts.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+import subprocess
+
+from redistricting.models import ComputedDistrictScore, Plan, ScoreFunction, Subject
+
+
+class Command(BaseCommand):
+    """
+    A one-off command to reconfigure the app to accomodate new config for
+    a second legislative body with 17 districts (while still preserving all
+    templates and plans).
+    """
+    help = 'Reconfigure for legislative body with 17 districts'
+
+    def handle(self, *args, **options):
+        """
+        Performs the steps
+        """
+        print('Reconfiguring for legislative body with 17 districts...')
+        remove_score_config()
+        download_pa_17_shapefile()
+        configure_templates()
+        print('Reconfiguration complete!')
+
+def remove_score_config():
+    """
+    Call removescoreconfig management command
+    """
+    print('Calling removecoreconfig...')
+    subprocess.check_call('./manage.py removescoreconfig', shell=True)
```

Comment:
Not too important, but just wanted to highlight that [`call_command`](https://docs.djangoproject.com/en/2.1/ref/django-admin/#running-management-commands-from-your-code) can be used to execute other Django management commands vs. using `subprocess`.